### PR TITLE
Update UI of order return-page #7595

### PR DESCRIPTION
**Pull request by gautampanday**

### CODE REVIEW

**Comment by gautampanday** [comment]
return item lie nielsen toolworks

**Comment by gautampanday** [comment]
return 2lie nielsen toolworks

File: templates/webshop/complaint.jinja [hunk 1]
```diff
@@ -6,77 +6,94 @@
...
+        <tbody>
+          <tr>
+            <td>
+              <img
+              src="{{ CDN }}{{ line.product.default_image.transform_command().thumbnail(200, 200, 'a') }}"
+              alt="{{ line.product.name }}"
+              title="{{ line.product.name }}"
+              class="img-responsive img-rounded margin-auto"
+              />
+              </td>
```
Comment (tarunbhardwaj) [inline templates/webshop/complaint.jinja]: indentation

File: templates/webshop/complaint.jinja [hunk 2]
```diff
@@ -6,77 +6,94 @@
...
-        <h4>Reason for Return</h4>
-      </div>
-      <div class="col-md-8 margin-top-15">
-        {{ complaint.return_reason.name }}
+        <h4>Reason for Return:<small> {{ complaint.return_reason.name }}
         {% if complaint.return_reason.description %}
           <br /><br />
           {{ complaint.return_reason.description }}
-        {% endif %}
+        {% endif %}</small></h4>
```
Comment (tarunbhardwaj) [inline templates/webshop/complaint.jinja]: ?

**Comment by tarunbhardwaj** [comment]
LGTM

**Comment by tarunbhardwaj** [comment]
LGTM

**Pull request merged**